components/SidePage/Suggestions/Suggestions.jsx

```diff
@@ -15,13 +16,20 @@ const SuggestItem = (suggest) => {
 
     const { asPath } = useRouter()
     const currentSection = asPath.split('/').filter(Boolean).at(1)
-    const link = `${origin}/${suggest?.suggest?.link}`
-    const leftText = suggest?.suggest?.text?.left
-    const target = suggest?.suggest?.text?.target
-    const rightText = suggest?.suggest?.text?.right
+    const link = `${origin}/${suggest?.link}`
+    const leftText =
+        suggest?.text?.left.length >= 3 ? '...' + suggest?.text?.left : suggest?.text?.left
```

Comment:
**question** it is better to request such a case on the backend (always putting a three-dot seems to be the wrong way)